python_modules/dagster/dagster/_core/definitions/asset_reconciliation_sensor.py

```diff
@@ -752,7 +752,7 @@ def build_asset_reconciliation_sensor(
     - This sensor has never tried to materialize it and it has never been materialized.
     - Any of its parents have been materialized more recently than it has.
     - Any of its parents are unreconciled.
-    - It is not currently up to date with respect to its FreshnessPolicy
+    - It is not currently up to date with respect to its FreshnessPolicy.
```

Comment:
Should this be `freshness policy`? The example asset graph uses this formatting. I'm not sure if this instance refers to the name of a specific property/attribute, or just a freshness policy in general.